runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -491,7 +548,7 @@ def StartDebug_prompt(dict: dict<any>)
   prompt_setcallback(promptbuf, function('PromptCallback'))
   prompt_setinterrupt(promptbuf, function('PromptInterrupt'))
 
-  if vvertical
+  if vvertical == true
```

Comment:
When using a Boolean variable, there is no need to it compare against `true` or `false`.  The previous code is more readable.

Reply:
Correct. I get burned once with Python due to the truthy/falsy thing that I now tend to be explicit on pretty much everything. However, being Boolean variables, I agree that the == true and ==false are not needed. I'll fix it asap.